static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-control.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+export const acquisitionsEpicControlTemplate = ({
+    copy,
+    componentName,
+    buttonTemplate,
+    testimonialBlock = '',
+    epicClass = '',
+    wrapperClass = '',
+}: {
+    copy: AcquisitionsEpicTemplateCopy,
+    componentName: string,
+    buttonTemplate: string,
+    testimonialBlock?: string,
+    epicClass?: string,
+    wrapperClass?: string,
+}) =>
+    `<div class="contributions__epic ${epicClass}" data-component="${componentName}">
+        <div class="${wrapperClass}">
+            <div>
+                <h2 class="contributions__title contributions__title--epic">
+                    ${copy.heading ? copy.heading : ''}
```

Comment:
you should be able to do some nested destructuring here to avoid this, i.e.

```javascript
export const acquisitionsEpicControlTemplate = ({
    copy: {
        heading = '',
        p1,
        p2
    },
    componentName,
    buttonTemplate,
    testimonialBlock = '',
    epicClass = '',
    wrapperClass = '',
}: {
    copy: AcquisitionsEpicTemplateCopy,
    componentName: string,
    buttonTemplate: string,
    testimonialBlock?: string,
    epicClass?: string,
    wrapperClass?: string,
})
```